### PR TITLE
EF Core 2.2 what's new and query tags

### LINKED ISSUE #1113: EF Core 2.2: query tags
Need to document the new feature. Working on a PR right now.

### LINKED ISSUE #1114: EF Core 2.2: what's new page
Working on a PR.

### PR DESCRIPTION
**Pull request by divega**
Fixes #1114: what's new page
Fixes #1113: query tags

### CODE REVIEW

File: entity-framework/core/what-is-new/ef-core-2.2.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+And you can execute database queries based on spatial data and operations:
+
+``` csharp
+  var nearestFriends =
+      (from f in context.Friends
+      orderby f.Location.Distance(myLocation) descending
+      select f).Take(5).ToList();
+```
+
+For more information on this feature, see the [spatial types documentation](xref:core/tbd/spatial).
```
Comment (divega) [inline entity-framework/core/what-is-new/ef-core-2.2.md]: need to update xref once #1096 is merged.

File: entity-framework/core/what-is-new/index.md [hunk 2]
```diff
@@ -13,13 +13,14 @@ You can use the links below to learn about new features in each release:
 ## Future releases
 
 - [EF Core Roadmap](xref:core/what-is-new/roadmap)
+- [EF Core 2.2](xref:core/what-is-new/ef-core-2.2)
```
Comment (divega) [inline entity-framework/core/what-is-new/index.md]: Need to move to recent releases once we RTM.

File: entity-framework/core/what-is-new/roadmap.md [hunk 3]
```diff
@@ -11,52 +11,56 @@ uid: core/what-is-new/roadmap
...
+5. **What is the synergy of the feature when used in combination with other products?** We favor features that enable or significantly improve the experience of using EF Core with other products, such as .NET Core, the latest version of Visual Studio, Microsoft Azure, etc.
 
-6. **What are the capabilities of the people available to work on a feature, and how to best leverage these resources?** Each member of the EF team and even our community contributors have different levels of experience in different areas and we have to plan accordingly. Even if we wanted to have “all hands on deck” to work on a specific feature like GroupBy translations, or many-to-many, that wouldn’t be practical.
+6. **What are the skills of the people available to work on a feature, and how to best leverage these resources?** Each member of the EF team and our community contributors have different levels of experience in different areas, so we have to plan accordingly. Even if we wanted to have "all hands-on deck" to work on a specific feature like GroupBy translations, or many-to-many, that wouldn't be practical.
```
Comment (bricelam) [inline entity-framework/core/what-is-new/roadmap.md]: No hyphen in “hands-on”. Only for adjectives like hands-on training.

File: entity-framework/toc.md [hunk 4]
```diff
@@ -12,24 +12,38 @@
...
+##### [Existing Database](core/get-started/aspnetcore/existing-db.md)
 ##### [⤤ EF Core and Razor Pages](/aspnet/core/data/ef-rp/intro)
 #### [Universal Windows Platform (UWP)](core/get-started/uwp/index.md)
-##### [UWP - New Database](core/get-started/uwp/getting-started.md)
+##### [New Database](core/get-started/uwp/getting-started.md)
+#### [.NET Framework](core/get-started/full-dotnet/index.md)
+##### [New Database](core/get-started/full-dotnet/new-db.md)
+##### [Existing Database](core/get-started/full-dotnet/existing-db.md)
+
+### Fundamentals
```
Comment (bricelam) [inline entity-framework/toc.md]: 👍

File: entity-framework/core/what-is-new/ef-core-2.2.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+
+## Spatial data support
+
+Spatial data can be used to represent the physical location and shape of objects.
+Many databases can natively store, index, and query spatial data. 
+Common scenarios include querying for objects within a given distance, and testing if a polygon contains a given location.
+EF Core 2.2 now supports working with spatial data from various databases using types from the [NetTopologySuite](https://github.com/NetTopologySuite/NetTopologySuite) (NTS) library.
+
+Spatial data support is implemented as a series of provider-specific extension packages.
+Each of these packages contributes mappings for NTS types and methods and the corresponding spatial types and functions in the database.
```
Comment (divega) [inline entity-framework/core/what-is-new/ef-core-2.2.md]: NRS: add a comma

File: entity-framework/toc.md [hunk 6]
```diff
@@ -12,24 +12,38 @@
 
 ### [What's New](core/what-is-new/index.md)
 #### [EF Core Roadmap](core/what-is-new/roadmap.md)
-#### [EF Core 2.1](core/what-is-new/ef-core-2.1.md)
+#### [EF Core 2.2 (in preview)](core/what-is-new/ef-core-2.2.md)
+#### [EF Core 2.1 (latest stable release)](core/what-is-new/ef-core-2.1.md)
 #### [EF Core 2.0](core/what-is-new/ef-core-2.0.md)
 #### [EF Core 1.1](core/what-is-new/ef-core-1.1.md)
 #### [EF Core 1.0](core/what-is-new/ef-core-1.0.md)
+#### [Upgrading from previous versions]
```
Comment (divega) [inline entity-framework/toc.md]: Should not have square brackets

**Pull request merged**